IdentityServer/v6/docs/content/quickstarts/4_ef.md

```diff
@@ -4,117 +4,193 @@ date: 2020-09-10T08:22:12+02:00
 weight: 5
 ---
 
+Welcome to Quickstart 4 for Duende IdentityServer! In this quickstart you will
+move configuration and other temporary data into a database using Entity
+Framework. 
+
 {{% notice note %}}
-For any pre-requisites (like e.g. templates) have a look at the [Quickstarts Overview]({{< ref "0_overview" >}}) first.
-{{% /notice %}}
 
-In the previous quickstarts, we created our client and scope data in code.
-On startup, IdentityServer loaded this configuration data into memory.
-If we wanted to modify this configuration data, we had to stop and start IdentityServer.
+We recommend you do the quickstarts in order, but if you'd like
+to start here, begin from a copy of [Quickstart 2's source code]({{< param
+qs_base >}}/2_InteractiveAspNetCore). You will also need to [install the IdentityServer
+templates]({{< ref "0_overview#preparation" >}}).
```

Comment:
IS QS2 the right base? QS3 has API access from the UI client, right?

Reply:
Thanks, this should have been QS3.